specification/components/schemas/responses/changeShortlistAndSendForTriage/200Response.yaml

```diff
@@ -0,0 +1,19 @@
+description: Referral updated with new service and sent for Triage
+headers:
+  x-correlation-id:
+    $ref: '../../headers/response/CorrelationID.yaml'
+  x-request-id:
+    $ref: '../../headers/response/RequestID.yaml'
+  ETag:
+    $ref: '../../headers/response/ETag.yaml'
+content:
+  application/fhir+json:
+    schema:
+      $ref: '../../eRS-ReferralRequest.yaml'
```

Comment:
As mentioned above I think the `ChangeShortlistAndSendForTriage` endpoint should return a `Parameters` resource which includes both the updated `ReferralRequest` and newly created `Appointment` resource.